asustor_main.c

```diff
@@ -190,6 +192,53 @@ static struct gpiod_lookup_table asustor_as6706_gpio_leds_lookup = {
 	},
 };
 
+static struct gpiod_lookup_table asustor_6300_gpio_leds_lookup = {
+	.dev_id = "leds-gpio",
+	.table = {
+		GPIO_LOOKUP_IDX(GPIO_IT87,   29, NULL,  0, GPIO_ACTIVE_HIGH), // power:front_panel
+		GPIO_LOOKUP_IDX(GPIO_IT87,   59, NULL,  1, GPIO_ACTIVE_HIGH), // power:lcd
+		GPIO_LOOKUP_IDX(GPIO_IT87,   56, NULL,  2, GPIO_ACTIVE_LOW),  // blue:power
+		GPIO_LOOKUP_IDX(GPIO_IT87,    8, NULL,  3, GPIO_ACTIVE_LOW),  // red:power
+		GPIO_LOOKUP_IDX(GPIO_IT87,   31, NULL,  4, GPIO_ACTIVE_LOW),  // green:status
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 20, NULL,  5, GPIO_ACTIVE_HIGH), // red:status
+		// 6
+		GPIO_LOOKUP_IDX(GPIO_IT87,   21, NULL,  7, GPIO_ACTIVE_LOW),  // green:usb
+		GPIO_LOOKUP_IDX(GPIO_IT87,   55, NULL,  8, GPIO_ACTIVE_HIGH), // blue:lan
+
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 23, NULL,  9, GPIO_ACTIVE_HIGH), // sata1:green:disk	++
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 24, NULL, 10, GPIO_ACTIVE_HIGH), // sata1:red:disk 	++
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 22, NULL, 11, GPIO_ACTIVE_HIGH), // sata2:green:disk	++
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 21, NULL, 12, GPIO_ACTIVE_HIGH), // sata2:red:disk 	++
+		{}
+		// GPIO_LOOKUP_IDX(GPIO_IT87,   53, NULL, 15, GPIO_ACTIVE_HIGH), // beep:status <<-- It can beep too!!!
+	},
+};
+
+static struct gpiod_lookup_table asustor_5304_gpio_leds_lookup = {
+	.dev_id = "leds-gpio",
+	.table = {
+		GPIO_LOOKUP_IDX(GPIO_IT87,   29, NULL,  0, GPIO_ACTIVE_HIGH), // power:front_panel
+		GPIO_LOOKUP_IDX(GPIO_IT87,   59, NULL,  1, GPIO_ACTIVE_HIGH), // power:lcd
+		GPIO_LOOKUP_IDX(GPIO_IT87,   56, NULL,  2, GPIO_ACTIVE_LOW),  // blue:power
+		GPIO_LOOKUP_IDX(GPIO_IT87,    8, NULL,  3, GPIO_ACTIVE_LOW),  // red:power
+		GPIO_LOOKUP_IDX(GPIO_IT87,   31, NULL,  4, GPIO_ACTIVE_LOW),  // green:status
+		GPIO_LOOKUP_IDX(GPIO_AS5304, 20, NULL,  5, GPIO_ACTIVE_HIGH), // red:status
+		//GPIO_LOOKUP_IDX(GPIO_IT87,   21, NULL,  6, GPIO_ACTIVE_LOW),  // blue:usb
+		GPIO_LOOKUP_IDX(GPIO_IT87,   21, NULL,  7, GPIO_ACTIVE_LOW),  // green:usb
+		GPIO_LOOKUP_IDX(GPIO_IT87,   55, NULL,  8, GPIO_ACTIVE_HIGH), // blue:lan
+
+		GPIO_LOOKUP_IDX(GPIO_AS5304, 1,  NULL,  9, GPIO_ACTIVE_HIGH), // sata1:green:disk	++
```

Comment:
This line and below seem weird, `[tab]++`?
```suggestion
		GPIO_LOOKUP_IDX(GPIO_AS5304, 1,  NULL,  9, GPIO_ACTIVE_HIGH), // sata1:green:disk
```

Reply:
I don't know (same for the previous comment), these parts are just copy&pasted from https://github.com/mafredri/asustor-platform-driver/discussions/37 - maybe @Tki2000 can explain the meaning?